R/metadata.R

```diff
@@ -46,10 +54,17 @@ setClass(
     load.rate.units="character",
     # Other
     station="character",
+    wq.sta.id="character",
+    station.lat="ANY",
+    station.lon="ANY",
+    disch.sta.id="character",
+    disch.basin.area="ANY",
+    wq.basin.area="ANY",
```

Comment:
again, requiring numeric would be nice. also please put the required units for area (both disch.basin.area and wq.basin.area) in their param descriptions in the roxygen2 header for `metadata()`